.github/workflows/continuous-integration.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7,3.8;3.9,'3.10']
+        python-version: [3.7,3.8,3.9,'3.10']
```

Comment:
Coding style: could you please add an empty space after the commas?

Reply:
fixed 75bda53